packages/chat-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -68,7 +68,10 @@ export const createDefaultChatHandlers = memoizeOne(
         }
       ) => {
         const updatedMetadata = options?.metadata ? { ...options.metadata } : {};
-        updatedMetadata.fileSharingMetadata = JSON.stringify(options?.attachmentMetadata || []);
+        if (options?.attachmentMetadata) {
```

Comment:
Should we also check if the array is not empty, just in case?

Reply:
Thanks! That's a great call.